app/web/src/components/FuncEditor/AttributeBindings.vue

```diff
@@ -200,8 +200,9 @@ const prototypeViews = computed(() =>
 
       const args = proto.prototypeArguments.map((arg) => ({
         name: funcArgumentsIdMap?.value[arg.funcArgumentId]?.name ?? "none",
-        // TODO(nick): re-evaluate this when prop id is added back to the attribute prototype argument bag.
-        prop: arg.inputSocketId ?? "none",
+        prop: arg.propId
+          ? funcStore.propIdToSourceName(arg.propId) ?? "none"
+          : "none",
```

Comment:
How this worked in the old engine: this was the "implicit internal provider to prop name" converter.